DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/CourseCompletionViewModelTests.cs

```diff
@@ -134,6 +134,83 @@ public void CourseCompletion_should_floor_round_percentageTutorialsCompleted()
             courseCompletionViewModel.PercentageTutorialsCompleted.Should().Be(66);
         }
 
+        [Test]
+        public void CourseCompletion_should_not_show_DiagnosticScore_when_null()
+        {
+            // Given
+            var expectedCourseCompletion = CourseCompletionHelper.CreateDefaultCourseCompletion(
+                diagnosticScore: null
+            );
+
+            // When
+            var courseCompletionViewModel = new CourseCompletionViewModel(config, expectedCourseCompletion, ProgressId);
+
+            // Then
+            courseCompletionViewModel.ShowDiagnosticScore.Should().BeFalse();
+        }
+
+        [Test]
+        public void CourseCompletion_should_not_show_DiagnosticScore_when_attempts_are_0()
+        {
+            // Given
+            var expectedCourseCompletion = CourseCompletionHelper.CreateDefaultCourseCompletion(
+                diagnosticScore: 0,
+                diagnosticAttempts: 0
+            );
+
+            // When
+            var courseCompletionViewModel = new CourseCompletionViewModel(config, expectedCourseCompletion, ProgressId);
+
+            // Then
+            courseCompletionViewModel.ShowDiagnosticScore.Should().BeFalse();
+        }
+
+        [Test]
+        public void CourseCompletion_should_show_DiagnosticScore_when_attempts_are_not_0()
+        {
+            // Given
+            var expectedCourseCompletion = CourseCompletionHelper.CreateDefaultCourseCompletion(
+                diagnosticScore: 10,

```

Comment:
Minor but it might be worth making this 0 here or not 0 in the test above. As at the moment an implementation which shows the score when it's not 0 would pass these tests 🙂

Reply:
Yep! Done in 08ae966 🙂